package.json

```diff
@@ -48,10 +48,15 @@
     "simple-oauth2": "^4.3.0"
   },
   "devDependencies": {
+    "@types/node": "^16.11.53",
+    "autoprefixer": "^10.4.8",
     "eslint": "^8.19.0",
     "eslint-config-next": "^12.2.0",
     "husky": "^8.0.1",
     "lint-staged": "^13.0.3",
-    "prettier": "^2.7.1"
+    "postcss": "^8.4.16",
+    "prettier": "^2.7.1",
+    "tailwindcss": "^3.1.8",
+    "typescript": "^4.7.4"
```

Comment:
New `devDependencies` are related to TypeScript (`@types/node` and `typescript`) or related to Tailwind CSS (`autoprefixer`, `postcss`, and `tailwindcss`).